src/core/transport/chttp2_transport.c

```diff
@@ -521,7 +521,6 @@ static void destroy_stream(grpc_exec_ctx *exec_ctx, grpc_transport *gt,
                                            s->global.id) == NULL);
   }
 
-  grpc_chttp2_list_remove_writable_stream(&t->global, &s->global);
```

Comment:
We used to not hold a ref while streams were in the writable list.
=> A stream may be destroyed while in the writable list.
=> We might need some cleanup here.

Now we hold a ref, so it's impossible to get here while the stream is flagged writable, so there's no need for the cleanup.